### PR TITLE
The squelch-crash option discussed on Telegram earlier

### PR DESCRIPTION
**Pull request by forensicmike**
And a slight update to the README.md for would be devs :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
...
+
+**You do not need to build this repo** in order to make changes (for a pull request, or for local development). Simply set your  PYTHONPATH environment variable to wherever you've clone this repo to. 
+
+For example, on Windows, assuming you clone to c:\src:
+
+    git clone https://github.com/frida/frida-tools.git
+    cd frida-tools
+    SET PYTHONPATH=c:\src\frida-tools
+
+Now when you run frida.exe, these scripts and any changes you make will be referenced instead!
```
Comment (oleavr) [inline README.md]: Would be good to wrap lines at 80 cols.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,15 @@
 # Frida CLI tools
 
 CLI tools for [Frida](http://www.frida.re).
+
+### Making dev changes to frida-tools
+
+**You do not need to build this repo** in order to make changes (for a pull request, or for local development). Simply set your  PYTHONPATH environment variable to wherever you've clone this repo to.
```
Comment (oleavr) [inline README.md]:
```suggestion
**You do not need to build this repo** in order to make changes (for a pull request, or for local development). Simply set your PYTHONPATH environment variable to wherever you've cloned this repo to. 
```

File: README.md (hunk 1 above)
Comment (forensicmike) [reply README.md]: 👍

File: README.md (hunk 2 above)
Comment (forensicmike) [reply README.md]: 👍

**Comment by forensicmike** [comment]
done, thanks for the fast review!

**Comment by oleavr** [comment]
Thanks! Just need to wrap lines in README.md so none of them exceed 80 cols.

File: README.md [hunk 3]
```diff
@@ -1,3 +1,20 @@
...
 
 CLI tools for [Frida](http://www.frida.re).
+
+### Making dev changes to frida-tools
+
+**You do not need to build this repo** in order to make changes (for a pull 
+request, or for local development).
+
+Simply set your PYTHONPATH environment variable to wherever you've cloned
+this repo to.
```
Comment (oleavr) [inline README.md]:
```suggestion
this repo to.
```

File: README.md [hunk 4]
```diff
@@ -1,3 +1,20 @@
...
+
+### Making dev changes to frida-tools
+
+**You do not need to build this repo** in order to make changes (for a pull 
+request, or for local development).
+
+Simply set your PYTHONPATH environment variable to wherever you've cloned
+this repo to. 
+
+For example, on Windows, assuming you clone to c:\src:
```
Comment (oleavr) [inline README.md]:
```suggestion
For example, on Windows, assuming you clone to `C:\src`:
```

File: README.md [hunk 5]
```diff
@@ -1,3 +1,20 @@
...
+
+    git clone https://github.com/frida/frida-tools.git
+    cd frida-tools
+    SET PYTHONPATH=c:\src\frida-tools
```
Comment (oleavr) [inline README.md]:
```suggestion
    SET PYTHONPATH=C:\src\frida-tools
```

**Comment by forensicmike** [comment]
My bad, should be good now

**Pull request merged**